src/index.js

```diff
@@ -0,0 +1,109 @@
+const API_KEY = 'http://localhost:5000';
```

Comment:
Nice job using a constant variable to reference a string which makes your code more readable and also in the long run more maintainable because we can look at the top of the file for the variable to update the string (like for example, if the port number changes and we have to update the endpoint). Then we could avoid combing through lots of code to make a small update.

Technically, the string referred to by this variable is not an API key, but an base endpoint. 
```suggestion
const BASE_URL = 'http://localhost:5000';
```